src/com/com_nrf24l01_types.hpp

```diff
@@ -0,0 +1,124 @@
+#ifndef COM_COM_NRF24L01_TYPES_HPP_
+#define COM_COM_NRF24L01_TYPES_HPP_
+
+namespace com {
+
+#define ON_ERROR_RETURN(x)                               \
```

Comment:
This isn't a type, I think this would justify a com_nrf...._error_handler file

Reply:
Factored out in own file.